src/ui-kit/components/Avatar/Avatar.styled.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+import { Avatar } from '@mui/material';
+
+const sizes = {
```

Comment:
Maybe, but if you have, for example, not 55x55 but 55x65, what will be two objects?
Please look again at the example you were given:
`const sizePicker = {
 sm:{
 width: '55 px',
 height: '55 px'},
 ....}`
 And then it shouldn't be like this : `width: ${(p) => sizes[p.size].width}`